resources/js/features/home/components/+root/FrontPageNews/FrontPageNews.tsx

```diff
@@ -0,0 +1,114 @@
+import { useLaravelReactI18n } from 'laravel-react-i18n';
+import type { FC } from 'react';
+
+import { HomeHeading } from '../../HomeHeading';
+import { SeeMoreLink } from '../../SeeMoreLink';
+
+// TODO track click and position when clicked
+
+export const FrontPageNews: FC = () => {
+  const { t } = useLaravelReactI18n();
+
+  return (
+    <div className="flex flex-col gap-2.5">
+      <HomeHeading>{t('News')}</HomeHeading>
+
+      <div className="grid grid-cols-2 gap-4 sm:flex sm:flex-col sm:gap-1">
+        <NewsCard
+          authorDisplayName="SporyTike"
+          href="#"
+          imageSrc="https://media.retroachievements.org/Images/103356.jpg"
+          postedAt="2d ago"
+          title="New Guide: Dragon Warrior II"
+          lead="A new achievement guide has been published for Dragon Warrior II, written by Nepiki! Continue the Quest and fulfill the legend."
+          tagLabel="Guide"
+        />
```

Comment:
I'm not fond of the gradient shadow on the images for these. I can't find the component in this PR, so I'm not sure if it's something we can change.

Reply:
Looks like I accidentally forgot to turn that off. Updated in latest.

It only makes sense if there's a tag in the image corner, like this:

![Screenshot 2024-10-27 at 11 40 41 AM](https://github.com/user-attachments/assets/629ff858-ba2e-406a-bbc3-a39930888a65)

It's coming from `<NewsCard />`.

Helpful tip: VSCode has very good built-in React editing and debugging support. To trace any React component, on Windows, hold down the Control key and then click on the component name in the template. VSCode will take you right to the component's definition.